server/text_generation_server/models/__init__.py

```diff
@@ -342,22 +342,19 @@ def get_model(
     model_type = config_dict.get("model_type", None)
 
     quantization_config = config_dict.get("quantization_config", None)
-    compression_config = config_dict.get("compression_config", None)
```

Comment:
@danieldk config renamed to quantisation config.

Reply:
I think we should check for both keys, at least for the time being. Some customers/users may have checkpoints that still have `compression_config`. Maybe with a comment that `compression_config` is for backwards compatibility?